Firmware/mmu2.h

```diff
@@ -286,6 +300,7 @@ class MMU2 {
     
     bool inAutoRetry;
     uint8_t retryAttempts;
+    uint16_t toolchange_counter;
```

Comment:
Total toolchange count is 32bit in eeprom while this variable is only 16bit. That assumes there will be no more than 64K toolchanges in a single print ... on MK3 this is probably a realistic assumption...